lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

```diff
@@ -1044,6 +1044,166 @@ func TestGetSubscriber(t *testing.T) {
 	tests.RunUnitTest(t, e, tc)
 }
 
+func TestGetSubscriberByExactIMSI(t *testing.T) {
+	configuratorTestInit.StartTestService(t)
+	deviceTestInit.StartTestService(t)
+	stateTestInit.StartTestService(t)
+	err := configurator.CreateNetwork(configurator.Network{ID: "n1"}, serdes.Network)
+	assert.NoError(t, err)
+
+	e := echo.New()
+	testURLRoot := "/magma/v1/lte/:network_id/subscribers/:subscriber_id"
+	handlers := handlers.GetHandlers()
+	getSubscriber := tests.GetHandlerByPathAndMethod(t, handlers, testURLRoot, obsidian.GET).HandlerFunc
+
+	// preseed 2 apns
+	apn1, apn2 := "foo", "bar"
+	_, err = configurator.CreateEntities(
```

Comment:
Create the apn through the rest api rather than through configurator

Reply:
Looked through the REST apis for this service as well as the other test cases, can't seem to find an appropriate api for creating an APN & the other test cases are consistently using the `configurator` for this purpose. Wondering if I should just stick to using the `configurator` then? @hcgatewood

Reply:
Yeah this is fine. There is a way, but this is easier and unrelated to the functionality under test